apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -861,7 +869,7 @@ static void utc_audio_pcm_writei_n(void)
 	ssize_t size;
 	char *buffer;
 
-	g_pcm = pcm_open(0, 0, PCM_IN, NULL);
+	g_pcm = pcm_open(0, 0, PCM_OUT, NULL);
 	TC_ASSERT_GT("pcm_writei", pcm_get_file_descriptor(g_pcm), 0);
```

Comment:
Why direction changed? just curious. Was it a mistake earlier?

Reply:
It was a mistake earlier. Fixed it now.